### PR TITLE
Bug: `requirements-ml` fix

### PR DESCRIPTION
**Pull request by taylorfturner**

### CODE REVIEW

File: requirements-ml.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 scikit-learn>=0.23.2
 keras>=2.4.3
 rapidfuzz>=2.6.1
-tensorflow-gpu>=2.6.4; sys.platform == 'linux'
+tensorflow-gpu <= 2.11; sys.platform == 'linux'
```
Comment (JGSweets) [inline requirements-ml.txt:4]: I think this needs to be the combo, right?

Comment (taylorfturner) [reply requirements-ml.txt:4]: touche

Comment (taylorfturner) [reply requirements-ml.txt]: fixed and pushed

**Review by ksneab7 — APPROVED** [review]

File: requirements-ml.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 scikit-learn>=0.23.2
 keras>=2.4.3
 rapidfuzz>=2.6.1
-tensorflow-gpu>=2.6.4; sys.platform == 'linux'
+tensorflow-gpu >=2.6.4, <= 2.11; sys.platform == 'linux'
```
Comment (JGSweets) [inline requirements-ml.txt:4]: I think it needs to be `<2.11`

Comment (taylorfturner) [reply requirements-ml.txt:4]: Not sure it does based on this: https://pypi.org/project/tensorflow-gpu/#history

Comment (JGSweets) [reply requirements-ml.txt:4]: nvm, 2.11 is the correct cutoff, you're correct.

**Review by JGSweets — DISMISSED** [review]

File: requirements-ml.txt (hunk 2 above)
Comment (taylorfturner) [reply requirements-ml.txt:4]: good double check

**Review by JGSweets — APPROVED** [review]

**Review by micdavis — APPROVED** [review]

**Pull request merged**